.gitignore

```diff
@@ -1,7 +1,41 @@
+*.xcuserstate
+*.xc*
+*.pbxproj
+*.out
 *.o
 *.mod
+*.smod
+*.xdmf
+.#*
+*.jpg
+*.pdf
+*.png
+*.svg
+*.swp
+*.dat
+*~
+sauve*
+incompact3d
 xcompact3d
+data/
+out/
+stats/
+planes/
+post/
+.DS_Store
+TAGS
+lint.txt
+\#*\#
+*.log
+docs/_build/
 build*
+Test*
+test*
```

Comment:
This is probably a bad idea because the CI is currently located in the folder `tests`, see [https://github.com/xcompact3d/Xcompact3d/tree/main/tests](https://github.com/xcompact3d/Xcompact3d/tree/main/tests)

Reply:
Ok we can remove these. Generally i was using Test/test to mark a folder where i was running a testing case.

Reply:
The workaround below should ignore all folders test* and should watch the folder tests ?
```
test*
!tests
```